lib/stm32/g4/Makefile

```diff
@@ -45,6 +45,10 @@ OBJS += pwr.o
 OBJS += rcc.o rcc_common_all.o
 OBJS += timer_common_all.o timer_common_f0234.o
 
+OBJS += usb.o usb_control.o usb_standard.o usb_msc.o
+OBJS += usb_hid.o
```

Comment:
looks fine, but someone remind me to add teh other classes there, see https://github.com/libopencm3/libopencm3/pull/1221 to add after we're all merged in....

Reply:
> For RCC: what's the significance of 24/48/96/170? Do we really need all of them? are any of them the max for any particular family/variant? On F1/f4, we actually had different maxes, or max to still use usb, so some of these made sense, but some of these seem like "just a random assortment" ? I'd like to only include useful configs out of the box,

It's a bit arbitrary, but I selected like so:
24MHz is in the low-power mode, supporting USB using PLL (26MHz is maximum)
48MHz is the lowest flash wait-state and supports USB using PLL.
96MHz is the highest that supports USB using PLL.
170MHz is the highest the chip supports, CRS is needed for USB.

I guess it's a bit arbitrary, and you can probably tell that we used that sequence before we had USB CRS working